Blahgger/Blahgger/Views/Account/Login.cshtml

```diff
@@ -0,0 +1,45 @@
+﻿@model Blahgger.Models.User
+
+@{
+    ViewBag.Title = "Login";
+}
+
+<h2>Login</h2>
+
+@using (Html.BeginForm())
+{
+    @Html.AntiForgeryToken()
```

Comment:
Don't forget to add the `ValidateAntiForgeryToken` attribute to your POST action method.